deploy/legionPipeline.groovy

```diff
@@ -93,9 +93,19 @@ def createjenkinsJobs(String commitID) {
 def runRobotTests(tags="") {
     withAWS(credentials: 'kops') {
     	withCredentials([file(credentialsId: params.Profile, variable: 'CREDENTIAL_SECRETS')]) {
-            env.tags=tags.toString().trim()
-            env.robot_tags= !env.tags.empty ? "-i " + env.tags.replaceAll(',',' -i ') : ""
-            env.nose_tags = !env.tags.empty ? "-a " + env.tags.replaceAll(',',' -a ') : ""
+            def tags_list=tags.toString().split(',')
+            def robot_tags= []
+            def nose_tags = []
+            for (item in tags_list) {
+                if (item.contains('-')) robot_tags.add(item)
```

Comment:
It looks hard to use

Reply:
Same here. Need to think over tags_list format.